signalilo/templates/deployment.yaml

```diff
@@ -27,10 +29,31 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+          env:
+          - name: SIGNALILO_UUID
+            value: {{ required ".Values.config.uuid is required" .Values.config.uuid | quote }}
+          - name: SIGNALILO_ICINGA_HOSTNAME
+            value: {{ required ".Values.config.icinga_hostname is required" .Values.config.icinga_hostname | quote }}
+          - name: SIGNALILO_ICINGA_URL
+            value:  {{ required ".Values.config.icinga_url is required" .Values.config.icinga_url | quote }}
+          - name: SIGNALILO_ICINGA_USERNAME
+            value: {{ required ".Values.config.icinga_username is required" .Values.config.icinga_username | quote }}
+          - name: SIGNALILO_ICINGA_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "signalilo.fullname" . }}
+                key: "icinga_password"
+          - name: SIGNALILO_ALERTMANAGER_PORT
+            value: {{ .Values.config.alertmanager_port | quote }}
+          - name: SIGNALILO_ALERTMANAGER_BEARER_TOKEN
```

Comment:
This probably also needs to go into the secret.

Reply:
Moved to secret